relay-server/src/actors/project.rs

```diff
@@ -380,6 +378,8 @@ mod limited_public_key_comfigs {
     #[serde(rename_all = "camelCase", remote = "PublicKeyConfig")]
     pub struct LimitedPublicKeyConfig {
         pub public_key: ProjectKey,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        pub numeric_id: Option<u64>,
```

Comment:
This is now equivalent to `PublicKeyConfig`, so you can remove the entire module.